<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>381</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_send_name"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_send">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tab_send">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="byteStr_2">
        <attribute name="title">
         <string>ByteStr</string>
        </attribute>
        <widget class="QLineEdit" name="edit_send_byte">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>351</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="UnicodeStr_2">
        <attribute name="title">
         <string>UnicodeStr</string>
        </attribute>
        <widget class="QLineEdit" name="edit_send_uni">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>351</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>381</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>  Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_recv_name"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_recv">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tab_recv">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="byteStr">
        <attribute name="title">
         <string>ByteStr</string>
        </attribute>
        <widget class="QLineEdit" name="edit_recv_byte">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>351</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="UnicodeStr">
        <attribute name="title">
         <string>UnicodeStr</string>
        </attribute>
        <widget class="QLineEdit" name="edit_recv_uni">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>351</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
